app/javascript/react/components/GameShowTile.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react'
+import ReviewsIndexContainer from './ReviewsIndexContainer'
 
 
 const GameShowTile = props => {
   const name = props.name
   const playerNum = props.playerNum
   const description = props.description
+  const reviews = props.reviews
 
```

Comment:
We can even destructure this further like so,
```
const GameShowTile = (props) => {
  const { name, playerNum, description, reviews } = props
```
which will create the same variables